README.md

```diff
@@ -523,6 +523,18 @@ var objectElement = minim.toElement({ foo: 'bar' });
 var keys = objectElement.keys() // ['foo']
 ```
 
+##### delete
+
+The `delete` method deletes a key from the `ObjectElement` instance.
```

Comment:
I think it's worth mentioning it's the key's content (value) and not the key (element) itself. This method does not accept the key only the keys value.